crashathon/settings.py

```diff
@@ -49,18 +53,15 @@
 # Database
 # https://docs.djangoproject.com/en/1.9/ref/settings/#databases
 
+DEFAULT_DATABASE = config(
+    'DATABASE_URL',
+    default='postgresql://crashathon:crashathon@localhost:5432/crashathon',
+    cast=dj_database_url.parse)
+
```

Comment:
nice, i forgot about this (and didn't mean to leave my username in the committed version :)